C++/main.cpp

```diff
@@ -23,6 +23,7 @@
 */
 #include <cstring>
 #include <iostream>
+#include <string>
```

Comment:
if we do this can we get rid of the `include <cstring>` above?

Reply:
Yes that seems to be true; done.